### PR TITLE
add models to admin

### PR DESCRIPTION
**Pull request by claudio-vagarinho**
Add current models to Django admin
Screenshot 2023-09-27 at 12 26 46

### CODE REVIEW

File: revent/settings.py [hunk 1]
```diff
@@ -142,12 +142,12 @@
...
-STORAGES = {
-    "default": {
-        "BACKEND": "storages.backends.s3.S3Storage",
-        "OPTIONS": {
-            "bucket_name": AWS_STORAGE_BUCKET_NAME,
-            "location": AWS_S3_ENDPOINT_URL,
-        },
-    },
-}
+# STORAGES = {
```
Comment (tiagocardoso98) [inline revent/settings.py:145]: What was the error? Was it a 403 error?
If so this [documentation](https://django-storages.readthedocs.io/en/1.5.2/backends/amazon-S3.html) might be helpful

Comment (claudio-vagarinho) [reply revent/settings.py]: Screenshot 2023-09-26 at 09 37 05

this is the issue I'm having

Comment (claudio-vagarinho) [reply revent/settings.py]: it was fixed, check the settings.py file

**Comment by kenvontucky** [comment]
@claudio-vagarinho please add some screenshots

**Review by tiagocardoso98 — APPROVED** [review]

**Review by kenvontucky — APPROVED** [review]

**Pull request merged**